tesla_powerwall/const.py

```diff
@@ -20,10 +20,14 @@ class Roles(Enum):
 
 class GridStatus(Enum):
     CONNECTED = "SystemGridConnected"
-    ISLANEDED_READY = "SystemIslandedReady"
-    ISLANEDED = "SystemIslandedActive"
+    ISLANDED_READY = "SystemIslandedReady"
+    ISLANDED = "SystemIslandedActive"
     TRANSITION_TO_GRID = "SystemTransitionToGrid"  # Used in version 1.46.0
+    TRANSITION_TO_ISLAND = "SystemTransitionToIsland"
```

Comment:
Note: I spotted this status being returned from my powerwall while running my integration test.  I've added it in both for completeness and because my tests will fail without either this change or an arbitrary wait 🤮